synapseclient/client.py

```diff
@@ -4253,7 +4658,7 @@ def _waitForAsync(self, uri, request, endpoint=None):
             uri + "/start", body=json.dumps(request), endpoint=endpoint
         )
 
-        # https://rest-docs.synapse.org/rest/org/sagebionetworks/repo/model/asynch/AsynchronousJobStatus.html
+        # <https://rest-docs.synapse.org/rest/org/sagebionetworks/repo/model/asynch/AsynchronousJobStatus.html>
```

Comment:
nit: comments in code won't make it to the docs pages so there isn't an additional need to wrap it in `<>`.

The only spot it might show up is in the source code, but it wouldn't make the hyperlink clickable.

Reply:
Will fix it before merging. Thanks so much!